src/mixer/playermanager.cpp

```diff
@@ -361,67 +358,73 @@ void PlayerManager::addConfiguredDecks() {
 
 void PlayerManager::addDeckInner() {
     // Do not lock m_mutex here.
-    QString group = groupForDeck(m_decks.count());
-    VERIFY_OR_DEBUG_ASSERT(!m_players.contains(group)) {
+    ChannelHandleAndGroup handle_group =
+            m_pEngine->registerChannelGroup(groupForDeck(m_decks.count()));
+    VERIFY_OR_DEBUG_ASSERT(!m_players.contains(handle_group.handle())) {
         return;
     }
 
-    int number = m_decks.count() + 1;
+    int index = m_decks.count() + 1;
```

Comment:
Can we revert that? I prefer number her. Index is for me likely 0 based which is not the case here.

Reply:
I understand the sentiment, but `number` is a very wide word, if you know the code it makes sense, but upon reading it it seemed a very generic nondescript name.

Reply:
At least "index" makes it worse, so let's stick with number until we have something better.

Reply:
53dac4575e